hmda/src/main/scala/hmda/validation/dsl/PredicateCommon.scala

```diff
@@ -64,6 +66,7 @@ object PredicateCommon {
 
   def empty[A]: Predicate[A] = (_: A) match {
     case s: String => s.isEmpty
+    case s: EthnicityEnum => s.code == 0
```

Comment:
Rather than modifying `empty` I believe that it would cleaner to use [equalTo](https://github.com/cfpb/hmda-platform/blob/master/hmda/src/main/scala/hmda/validation/dsl/PredicateCommon.scala#L7) in the code.